packages/server-side-render/src/server-side-render.js

```diff
@@ -60,12 +61,16 @@ export class ServerSideRender extends Component {
 			urlQueryArgs = {},
 		} = props;
 
+		const sanitizedAttributes =
+			attributes &&
+			__experimentalSanitizeBlockAttributes( block, attributes );
```

Comment:
Just thinking out loud: An alternative could be to use `createBlock` here. That way we wouldn't need to add any extra public functions to `@wordpress/blocks`.

```js
const sanitizedAttributes = createBlock( block, attributes ).attributes;
```

But I suppose it makes sense for `@wordpress/blocks` to have `sanitizeBlockAttributes` as it might be useful in other situations? What do you think?

Reply:
Clever! But I fear that it's semantically incorrect? In the future, we could add more logics to the `createBlock` factory than `sanitizeBlockAttributes` only, that would be hard to catch any unexpected side-effects here.

Reply:
Fair enough!